src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -202,6 +207,11 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
     --pf-c-toolbar--spacer: var(--pf-c-toolbar__item--m-bulk-select--spacer);
   }
 
+  // Expand
+  &.pf-m-expand-all.pf-m-expanded {
```

Comment:
do you need the `pf-m-expand-all` modifier if theres not really any styles attached ?

Reply:
I think so, `pf-m-expanded` is a state of the expand all element, not just any item. I'm following the convention of the other items that have a specific use (bulk-select, etc), or I would have made expand-all it's own element.

Reply:
sounds good